### ISSUE TITLE
Combine Links and Books in Shared Reading List

### ISSUE THREAD

**Issue by kjaymiller** [kjaymiller/shared-reading-list#26]
Do we really need to keep the books and links separated. I feel this is something that can be done at the database level and not at the base level.

**Comment by jcode89** [comment]
I don't believe they need to stay separate. We can tie in links to `class ReadingList` by way of new methods that gather what we need, if we even need it. 

Jonathan Colón
